.github/workflows/CMake-minimal-version.yaml

```diff
@@ -0,0 +1,53 @@
+name: CMake minimal version
+# The aim of this test derminate minimal supported CMake version for LibRS project
+# and check if it equlas to expected minimal CMAke version
+
+on:
+  push:
+  pull_request:
+  
+env:
+  EXPECTED_CMAKE_VERSION: "VERSION 3.8.0"
+
+jobs:
+  test_downloader:
+    name: "Run CMake compatibility tool"
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: "Get current CMake version"
+        id: cmake_version
+        run: |
+          sudo apt-get install  xorg-dev libglu1-mesa-dev
+          
+          # Go back from RealSense directory to parent directory  
+          cd ..
+          
+          git clone https://github.com/nlohmann/cmake_min_version.git --depth 1
+          # Go to specific last commit to porpose avoid using breacking changes in the repo.
```

Comment:
Does this work?
clone with --depth 1 and then checkout a specific commit?
Please change the comment to
`We clone a specific commit which we tested the process as working`

Reply:
It's not..
`fatal: not a git repository (or any of the parent directories): .git`
Remove the --depth 1

Reply:
--depth -1 removed. You right, After next commit it will not work.

Reply:
> Does this work? clone with --depth 1 and then checkout a specific commit? Please change the comment to `We clone a specific commit which we tested the process as working`

I think it worked because specified commit is last commit (--depth 1). Or did you copied one not last commit?

Reply:
You are right.